src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -250,5 +254,69 @@ def update_confg(config_path: str, new_autovalidator_address: str, new_codename:
         raise RuntimeError(f"Failed to write to the configuration file: {config_path}.\n Error: {e}")
 
 
+def get_commands_from_server(
+    subnet_identifier: str, subnet_realm: str, wallet: bt.wallet, autovalidator_address: str
+) -> list:
+    """
+    Request commands to the server
+    Args:
+        subnet_identifier: Subnet Identifier
+        subnet_realm: Subnet Realm
+        wallet: Bittensor wallet object
+        autovalidator_address: AutoValidator Address
+    Returns:
+        list: List of commands
+    Example:
+        request_commands_to_server("computehorde", "mainnet", wallet, "http://localhost:8000")
+    """
+    url = f"{autovalidator_address}/api/v1/commands/"
+    headers = {
+        "Note": "",
+        "SubnetID": subnet_identifier,
+    }
+    response = make_signed_request("GET", url, headers, "", wallet, subnet_realm)
+    if response.status_code == 200:
+        data = response.json()
+        print(data)
+        print("Commands successfully retrieved.")
+        return data
+    else:
+        print(f"Failed to get commands. Status code: {response.status_code}")
+        print(response.text)
+        return []
+
+
+def get_normalized_codename_from_server(
+    subnet_identifier: str, subnet_realm: str, wallet: bt.wallet, autovalidator_address: str
+) -> str:
+    """
+    Get the codename from the server
+    Args:
+        subnet_identifier: Subnet Identifier
+        subnet_realm: Subnet Realm
+        wallet: Bittensor wallet object
+        autovalidator_address: AutoValidator Address
+    Returns:
+        str: Codename
+    Example:
+        get_codename_from_server("12", "mainnet", wallet, "http://localhost:8000")
```

Comment:
```suggestion
        >>> get_codename_from_server("12", "mainnet", wallet, "http://localhost:8000")
        computehorde
```
examples, on the other hand, are good, though the traditional way to write them would be like above.